lib/dradis/plugins/calculators/dread/engine.rb

```diff
@@ -25,10 +21,17 @@ class Engine < ::Rails::Engine
     end
 
     initializer 'calculator_dread.mount_engine' do
-      Rails.application.routes.append do
-        mount Dradis::Plugins::Calculators::DREAD::Engine => '/', as: :dread_calculator
+      Rails.application.reloader.to_prepare do
+        if (ActiveRecord::Base.connection rescue false) && ::Configuration.table_exists?
```

Comment:
Why is this needed? I don't see it in other integrations?

Reply:
See: https://github.com/dradis/dradis-html_export/blob/main/lib/dradis/plugins/html_export/engine.rb#L26. This is causing timing issues during OVA building as this addong is loaded when the main app is initialized.

Reply:
Added comment for future reference ✅